src/logging.cpp

```diff
@@ -125,9 +125,9 @@ bool BCLog::Logger::WillLogCategory(BCLog::LogFlags category) const
 
 bool BCLog::Logger::WillLogCategoryLevel(BCLog::LogFlags category, BCLog::Level level) const
 {
-    // Log messages at Warning and Error level unconditionally, so that
-    // important troubleshooting information doesn't get lost.
-    if (level >= BCLog::Level::Warning) return true;
+    // Log the Info, Warning, and Error message levels unconditionally, so that
+    // important troubleshooting information isn't lost.
+    if (level >= BCLog::Level::Info) return true;
```

Comment:
I feel like this almost deserves its own PR (maybe you are already planning on doing so). afaict we don't log anything right now using `Info` which is the only reason this change is safe but that might not be clear to reviewers of this rather large PR. (If for example we had a lot of log locations using `Info` then they would all need to be revisited w.r.t. to disk filling attacks)

Reply:
Good point. My original thinking was that `info` would be conditional like `debug` and `trace` and that only `warning` and `error` would be unconditional, which would also reduce the volume of default logging. However, that changed during the discussions in https://github.com/bitcoin/bitcoin/pull/25306. Will keep your suggestion in mind while restructuring the update here.